### ISSUE TITLE
Waterfox Website shows Figma logo in tab

### ISSUE THREAD

**Issue by nevinpuri** [MrAlex94/Waterfox#2575]
### What happened?

Was scrolling through waterfox.net and I saw the Figma logo in the Waterfox tab. Not sure if this is intentional.

![image](https://user-images.githubusercontent.com/39220032/153890020-939c0a66-a226-4bf4-af2f-ff0990a6cfcc.png)

### Reproducible?

- [ ] I have tested this on Firefox.

### Version

G3.𝑋 (Current)

### What platform are you seeing the problem on?

Windows

**Comment by jakenvac** [comment]
I think the whole screenshot leaves a lot to be desired. I think it would be great to display something in the browsing window area or not show it at all. (Maybe fade it out just below the url bar or something.

**Comment by dirnb** [comment]
> I think the whole screenshot leaves a lot to be desired. I think it would be great to display something in the browsing window area or not show it at all. (Maybe fade it out just below the url bar or something.

Exactly. I think they should just cut the screenshot exactly where the black "# FEATURES" background begins and put a from the-bottom-up grey fade on the image so that it mainly highlights the URL bar.
